pages/blog/whats-new-in-graphql-cli-4.1.0.mdx

```diff
@@ -0,0 +1,65 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: "What's new in GraphQL CLI 4.1.0",
+  author: 'ephelan',
+  tags: ['react'],
+  date: '2020-10-12',
+  description: "An overview of what we've added in this release of GraphQL CLI",
+  image:
+    'https://user-images.githubusercontent.com/20847995/67651234-85bf1500-f916-11e9-90e5-cb3bd0e6a338.png',
+};
+
+export default withArticle({ ...meta });
+
+[GraphQL CLI](http://graphql-cli.com) is your one-stop shop for developing full-stack GraphQL applications in Node.js. Since the release of 4.0.0, we have added a number of enhancements which will further elevate your GraphQL development experience.
```

Comment:
```suggestion
[GraphQL CLI](http://graphql-cli.com) is your one-stop shop for developing full-stack GraphQL applications in Node.js. Since the release of 4.0.0, we have added a number of enhancements which will further elevate your GraphQL development experience.
```

We need something stronger I think.

Something like:

```
GraphQL-CLI is init, codegen and generate can help both begginers and advanced users. 
GraphQL-CLI aggregates multiple projects from community giving developers best getting started experience. Tools that we including in CLI are mature and developed over the years based on Guild's experience in pushing production ready GraphQL solutions and also collaboration with Red Hat community projects